refactor: adjust notifications empty state

dated opened the pull request:
<!--
Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!

Please make sure you're familiar with and follow the instructions in the [contributing guidelines](https://ark.dev/docs/program-incentives/guidelines/contributing).

Please fill out the information below to expedite the review and (hopefully) merge of your pull request!
-->

## Summary

https://app.clickup.com/t/gb247k

<!-- What changes are being made? -->

<!-- Why are these changes necessary? -->

<!-- How were these changes implemented? -->

## Checklist

<!-- Have you done all of these things?  -->

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

<!-- Feel free to add additional comments. -->

dated committed:
refactor: adjust notifications empty state

faustbrian commented:
Task linked: [CU-gb247k [Notifications] adjust empty state of notifications dropdown](https://app.clickup.com/t/gb247k)

boldninja committed:
chore: optimize 2 SVG(s)

Optimized SVG(s):
- src/app/assets/images/dark/empty-notifications.svg
- src/app/assets/images/empty-notifications.svg

dated committed:
chore: update snapshots
